src/otf_api/api/trends/trend_client.py

```
@@ -0,0 +1,42 @@
+from typing import Any
+
+from otf_api.api.client import API_GATEWAY_BASE_URL, OtfClient
+
+
+class TrendClient:
+    """Client for retrieving workout trends/stats data from the OTF Gateway API."""
+
+    def __init__(self, client: OtfClient):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)